extensions/quarkiverse-couchbase.yaml

```diff
@@ -0,0 +1,6 @@
+---
+group-id: "io.quarkiverse.couchbase"
+artifact-id: "quarkus-couchbase"
+versions:
```

Comment:
No need to specify versions, that will be read from Maven Central

Reply:
Ah ok, that's good, that step did seem a bit manual.  I'll remove.